spec/logstash/plugin_mixins/scheduler/rufus_impl_spec.rb

```diff
@@ -83,7 +83,10 @@
     let(:counter) { java.util.concurrent.atomic.AtomicLong.new(0) }
 
     before do
-      scheduler.cron('* * * * * *') { counter.increment_and_get; sleep 3.25 } # every second
+      scheduler.cron('* * * * * *') do # every second
+        counter.increment_and_get
+        sleep_at_least 3.25
```

Comment:
spec was failing occasionally (unrelated to the eager `::EtOrbi::EoTime.now` above), 
this + the lower frequency makes it more predictable